js/app.js

```diff
@@ -0,0 +1,42 @@
+function isValidCard(numCard) {
+  var arrayReverse = [];
+  var state = '';
+  var strNum = numCard.toString();
+
+  // Almacena el numero dado al reves
+  for (var i = strNum.length - 1; i >= 0; i--) {
+    arrayReverse.push(parseInt(strNum[i]));
+  }
+
+  // Modifica los valores pares y los modifica.
+  for (var i = 1; i < arrayReverse.length - 1; i++) {
```

Comment:
Este `for` irá de 1 al 14 (`i < 15`), por lo que no considera el último dígito. `i <= arrayReverse.length` servirá en este caso para recorrer el arreglo completo.
En general, si `i` comienza en 0, `i < array.length` o `i <= array.length - 1` es correcto para recorrer el entero. Si `i` comienza en 1, `i <= array.length` es correcto.